### PR TITLE
Improve garage tabs handling

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Garage reopens with previously selected tab
- Selected tab can be deselected by clicking the button again or left clicking
- Allow using `TAB` key to cycle between tabs

### CODE REVIEW

File: addons/garage/XEH_preInit.sqf [hunk 1]
```diff
@@ -6,6 +6,7 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+GVAR(currentTab) = -1;
```
Comment (CreepPork) [inline addons/garage/XEH_preInit.sqf]: By default, why shouldn’t it start at 0?

Comment (mharis001) [reply addons/garage/XEH_preInit.sqf]: I think I chose this since BI garage initially opens with no tab selected.

**Comment by mharis001** [comment]
Can't reproduce, working fine on my end after 20+ attempts.

**Comment by neilzar** [comment]
Happens every time for me, running only CBA and ZEN. Build using `build.py`.

**Comment by mharis001** [comment]
Determined to result from `unsafeCVL` changes in development branch. Unrelated to changes in this PR.

**Pull request merged**